webdriver/tests/accept_alert/accept.py

```diff
@@ -10,7 +9,7 @@ def accept_alert(session):
         "POST", "session/{session_id}/alert/accept".format(**vars(session)))
 
 
-def test_null_response_value(session, url):
+def test_null_response_value(session, url, inline):
```

Comment:
While touching this line can you please remove the `url` fixture? It's not needed here.